src/main/java/com/commercetools/sync/inventories/InventoryService.java

```diff
@@ -0,0 +1,69 @@
+package com.commercetools.sync.inventories;
+
+import io.sphere.sdk.channels.Channel;
+import io.sphere.sdk.commands.UpdateAction;
+import io.sphere.sdk.inventory.InventoryEntry;
+import io.sphere.sdk.inventory.InventoryEntryDraft;
+
+import javax.annotation.Nonnull;
+import java.util.List;
+import java.util.Set;
+import java.util.concurrent.CompletionStage;
+
+/**
+ * Interface contains methods to perform sphere queries/commands related to Inventory topic.
+ * Listed operations are used in {@link InventorySync}.
+ */
+interface InventoryService {
+
+    /**
+     * Fetches all {@link InventoryEntry} objects whose sku is included in {@code skus}.
+     * For some {@code skus} there could be no matching entries in CT platform, so for such skus no entries would be
+     * returned. Because of that result can contain inventory entries of skus, that are only subset of passed
+     * {@code skus}, or even in empty list when no matching entry is found.
+     * Returned entries have an expanded reference to the {@code supplyChannel}.
+     *
+     * @param skus {@link Set} of sku values, used in search predicate
+     * @return {@link List} of matching entries or empty list when there was no entry of sku matching to {@code skus}.
+     */
+    @Nonnull
```

Comment:
IMHO Nonnull annotations on methods here and in implementation are excessive. It's quite obvious that CompletionStage will be provided